src/storage/v2/storage.cpp

```diff
@@ -1966,4 +1998,39 @@ void Storage::SetIsolationLevel(IsolationLevel isolation_level) {
   isolation_level_ = isolation_level;
 }
 
+void Storage::RestoreReplicas() {
+  MG_ASSERT(memgraph::storage::ReplicationRole::MAIN == GetReplicationRole());
+  if (!ShouldStoreAndRestoreReplicas()) {
+    return;
+  }
+  spdlog::info("Restoring replicas.");
+
+  for (const auto &[replica_name, replica_data] : *kvstorage_) {
+    spdlog::info("Restoring replica {}.", replica_name);
+
+    const auto maybe_replica_status = replication::JSONToReplicaStatus(nlohmann::json::parse(replica_data));
+    if (!maybe_replica_status.has_value()) {
+      LOG_FATAL("Could parse previously saved configuration of replica {}.", replica_name);
```

Comment:
```suggestion
LOG_FATAL("Cannot parse previously saved configuration of replica {}.", replica_name);
```

Maybe "Couldn't" also good, I am not sure. Either way, please negate the verb.